src/components/MapMenuView.tsx

```diff
@@ -14,6 +14,12 @@ import { Icon } from 'react-native-elements';
 
 import { ItemProps, HeaderProps, MapMenuProps } from '../types/inventoryTypes';
 
+const NAV_HEIGHT = 90;
```

Comment:
Is this the height of the top bar with the logo? If the height is needed to be declared multiple times might be good to have a constants file